pyproject.toml

```diff
@@ -70,7 +70,7 @@ pycoin = "~0.92.20230326"
 requests = "=2.32.3"
 service_identity = "~21.1.0"
 pexpect = "~4.8.0"
-intervaltree = "~3.1.0"
+sortedcontainers = "~2.4.0"
```

Comment:
We don't need `intervaltree` anymore, but we do still use `sortedcontainers` which is an indirect dependency that should be made explicit regardless.